backend/src/main/java/com/htecgroup/skynest/service/impl/FileServiceImpl.java

```diff
@@ -126,6 +128,19 @@ private List<FileResponse> asFileResponseList(List<FileMetadataEntity> allFiles)
         .collect(Collectors.toList());
   }
 
+  @Override
+  @RecordAction(objectId = "[0].toString()", actionType = ActionType.DELETE)
+  public FileResponse deleteFile(UUID fileId) {
```

Comment:
Does every method that needs to trigger this annotation has the same input params?

Reply:
No, I intend to use this for every sort of action ( edits, downloads ...). That's why I need to send the actionType